HDF5Examples/config/cmake/jrunTest.cmake

```diff
@@ -21,7 +21,7 @@ if (NOT TEST_PROGRAM)
   message (FATAL_ERROR "Require TEST_PROGRAM to be defined")
 endif ()
 if (NOT TEST_LIBRARY_DIRECTORY)
-  message (STATUS "Require TEST_LIBRARY_DIRECTORY to be defined")
+  message (VERBOSE "Require TEST_LIBRARY_DIRECTORY to be defined")
```

Comment:
Does "Require TEST_LIBRARY_DIRECTORY to be defined" mean defining it is required and it will be defined later, or it must be defined and it isn't?

Reply:
Should be optional